core/src/test/scala/slamdata/AutoBumpTest.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Precog Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package precog
+
+import org.specs2.mutable.Specification
+
+class AutoBumpTest extends Specification {
```

Comment:
`AutoBumpSpec`

Reply:
Yeah. I'm letting IntelliJ create it, and it has a weird default. Well, perhaps not _weird_, just not well suited to Specs.